test/e2e/e2e_test.go

```diff
@@ -0,0 +1,201 @@
+// Package integration_test implements simple integration test against a local cluster, whose config is loaded from the
+// kubeconfig file.
+package e2e_test
+
+import (
+	"context"
+	"encoding/json"
+	"os"
+	"path"
+	"strings"
+	"testing"
+	"time"
+
+	sdkEvent "github.com/newrelic/infra-integrations-sdk/data/event"
+	"github.com/newrelic/nri-kube-events/pkg/events"
+	"github.com/newrelic/nri-kube-events/test/e2e"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/rand"
+	"k8s.io/client-go/informers"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
```

Comment:
I'd move `github.com/newrelic/nri-kube-events` imports as a third group.

Reply:
I just ran `goimports` here, would that be compatible with the linter?

Reply:
I think I usually follow https://github.com/daixiang0/gci linter, which is also available in golanci-lint.